data/strings/strings.txt

```diff
@@ -868,24 +868,276 @@
+    ca = • Sense anuncis, sense rastrejadors
+    cs = • Žádné reklamy, žádné sledování
+    da = • Ingen annoncer, ingen sporing
+    de = • Keine Werbung, kein Tracking
+    el = • Χωρίς διαφημίσεις, χωρίς παρακολούθηση
+    es = • Sin anuncios, sin rastreadores
+    et = • Pole reklaame, pole jälgimist
+    eu = • Ez iragarkirik, ez jarraipenik
+    fa = ﺕﺭﺎﻈﻧ ﻥﻭﺪﺑ ،ﯽﺑﺎﯾﺩﺭ ﻥﻭﺪﺑ ،ﺕﺎﻐﯿﻠﺒﺗ ﻥﻭﺪﺑ •
+    fi = • Ei mainoksia, ei seurantaa
+    fr = • Pas de publicité, pas de pistage
+    he = םינותנ ףוסיא אלל, בקעמ אלל, תועדומ אלל •
+    hu = • Nincs nyomkövetés, nincsenek hirdetések
+    id = • Tanpa ikla, tanpa pelacakan
+    it = • Nessuna pubblicità, nessun tracciamento
+    ja = • 広告なし、追跡なし、監視なし
+    ko = • 광고 없음, 추적 없음, 감시 없음
+    mr = • जाहिराती नाहीत, ट्रॅककिंग नाही, डेटा संग्रह नाही
+    nb = • Ingen annonser, ingen sporing
+    nl = • Geen advertenties, geen tracking
+    pl = • Brak reklam, brak śledzenia
+    pt = • Sem anúncios, sem rastreamento
+    pt-BR = • Sem anúncios, sem rastreamento
+    ro = • Fără reclame, fără urmărire
+    ru = • Без рекламы, без трекинга, без слежки
+    sk = • Žiadne reklamy, žiadne sledovanie
+    sv = • Inga annonser, ingen spårning
+    sw = • Hakuna matangazo, hakuna ufuatiliaji
+    th = • ไม่มีโฆษณา ไม่มีการติดตาม ไม่มีการรวบรวมข้อมูล
+    tr = • Reklam yok, izleme yok
+    uk = • Без реклами, без трекінгу, без стеження
+    vi = • Không quảng cáo, không theo dõi
+    zh-Hans = • 无广告、无跟踪、无数据收集
+    zh-Hant = • 無廣告、無跟踪、無數據收集
+
+  [about_proposition_2]
+    tags = android
+    comment = Text in About screen
+    en = • No battery drain, no background data
+    ar = ﺔﻳﺭﺎﻄﺒﻟﺍ ﻑﺰﻨﺘﺴﺗ ﻻ •
+    be = • Не разраджае батарэю
+    bg = • Не изтощава батерията
+    ca = • No esgota la bateria
+    cs = • Nevybíjí baterii
+    da = • Dræner ikke batteriet
+    de = • Entlädt die Batterie nicht
+    el = • Δεν αδειάζει την μπαταρία
+    es = • No agota la batería
+    et = • Ei tühjenda akut
+    eu = • Ez du bateria agortzen
+    fa = • ﺪﻨﮐ ﯽﻤﻧ ﯽﻟﺎﺧ ﺍﺭ ﯼﺮﺗﺎﺑ
+    fi = • Ei tyhjennä akkua
+    fr = • Ne vide pas la batterie
+    he = הללוסה תא ןקורמ אל •
+    hu = • Nem meríti le az akkumulátort
+    id = • Tidak menguras baterai
+    it = • Non scarica la batteria
+    ja = • バッテリーを消耗しない
+    ko = • 배터리를 소모하지 않습니다
+    mr = • बॅटरी काढून टाकत नाही
+    nb = • Tapper ikke batteriet
+    nl = • Trekt de batterij niet leeg
+    pl = • Nie drenuje baterii
+    pt = • Não descarrega a bateria
+    pt-BR = • Não descarrega a bateria
+    ro = • Nu consumă bateria
+    ru = • Не разряжает батарею
+    sk = • Nevybíja batériu
+    sv = • Laddar inte ur batteriet
+    sw = • Haimalizi betri
+    th = • ไม่ทำให้แบตเตอรี่หมด
+    tr = • Pili boşaltmaz
+    uk = • Не розряджає акумулятор
+    vi = • Không làm hao pin
+    zh-Hans = • 不会耗尽电池
+    zh-Hant = • 不會耗盡電池
+
+  [about_proposition_3]
+    tags = android
+    comment = Text in About screen
+    en = • Fast, minimalistic and not bloated
```

Comment:
```suggestion
    en = • Fast, minimalistic, and not bloated
```

Reply:
This also doesn't translate well by automated translations as seen in the results. Not sure what would be a better input.